src/py/flwr/common/exit/exit_code.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Exit codes."""
+
+
+from __future__ import annotations
+
+
+class ExitCode:
+    """Exit codes for Flower components."""
+
+    # System exit codes
+    GRACEFUL_EXIT = 10  # Graceful exit requested by the user
+
+    # SuperLink-specific exit codes (100-199)
+    SUPERLINK_THREAD_CRASH = 100
```

Comment:
@panh99 should we add the error code to the name here?
```suggestion
    EC100_SUPERLINK_THREAD_CRASH = 100
    # Or
    SUPERLINK_100_THREAD_CRASH = 100
```

Reply:
@danieljanes I don't think so. It's unusual to include numbers in codes, such as `grpc.StatusCode` or HTTP status codes.

Reply:
@panh99: HTTP status code [counterexample](https://github.com/encode/starlette/blob/master/starlette/status.py) in Starlette

We can merge as-is, but having the status code (as opposed to just the name) in the constant would make the code easier to read (all the places where we call `flwr_exit`)

Reply:
Hmmm, that's a good example. I am okay with having `EC<code>` as prefix. Just a bit worried if some names are too long. For example, `EC301_SUPERNODE_NODE_AUTH_KEYS_REQUIRED`.

Reply:
Plus, if we add `EC<code>` as prefix, would the prefixes `SUPERNODE`/`SUPERLINK`/`COMMON` be a bit redundant? Wdyt?